services/files/src/routes/file.ts

```diff
@@ -1,6 +1,39 @@
 import { asyncHandler } from "@api/common";
 import express from "express";
 
-import { FileModel } from "../models/file";
+import { uploadFile, getFileUrl, getDownloadUrl } from "src/storage";
 
 export const fileRoutes = express.Router();
+
+fileRoutes.route("/upload").post(
+  asyncHandler(async (req, res) => {
+    const { file } = req;
+
+    if (!req.user) {
+      throw new Error("User must be logged in");
```

Comment:
Instead of throwing a normal `Error`, I have created a few different errors under the `errors.ts` file in `@api/common`. One of these is `BadRequestError` which is used to say that the user submitted something wrong, and it helps with error handling. So can you change all of the `throw new Error` to `throw new BadRequestError`?